### PR TITLE
Add `use_Karras_sigmas` to LMSDiscreteScheduler

### PR DESCRIPTION
**Pull request by Isotr0py**
Discussion: #2905

This PR allows for the utilization of `use_karras_sigmas` in the LMSDiscreteScheduler, refers to the contribution in #3001.

**LMS Discrete Scheduler**
```python
from diffusers import StableDiffusionPipeline, LMSDiscreteScheduler
import torch 

seed = 33

pipe = StableDiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4").to("cuda")
pipe.scheduler = LMSDiscreteScheduler.from_config(pipe.scheduler.config, use_karras_sigmas=True)

prompt = "a Roman solider driving a car on mars"

generator = torch.Generator(device="cuda").manual_seed(seed)
image = pipe(prompt, generator=generator, num_inference_steps=20).images[0]

display(image)
```
sd

**K_diffusion - sample_lms**
```python
import torch
from diffusers import StableDiffusionKDiffusionPipeline

seed = 33

pipe = StableDiffusionKDiffusionPipeline.from_pretrained('CompVis/stable-diffusion-v1-4').to('cuda')
pipe.set_scheduler('sample_lms')

prompt = "a Roman solider driving a car on mars"

generator = torch.Generator(device="cuda").manual_seed(seed)
image = pipe(prompt, generator=generator, num_inference_steps=20, use_karras_sigmas=True).images[0]

display(image)
```
ksd

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Great PR, good job!
